WNPRC_EHR/resources/queries/study/waterSchedule.query.xml

```diff
@@ -0,0 +1,192 @@
+<query xmlns="http://labkey.org/data/xml/query">
+    <metadata>
+        <tables xmlns="http://labkey.org/data/xml">
+            <table tableName="waterSchedule" tableDbType="TABLE">
+                <tableUrl>/EHR/treatmentDetails.view?key=${lsid}</tableUrl>
+                <pkColumnName>primaryKey</pkColumnName>
+                <columns>
+                    <column columnName="primaryKey">
+                        <isHidden>true</isHidden>
+                        <isKeyField>true</isKeyField>
+                    </column>
+                    <column columnName="primaryKey2">
```

Comment:
@dnicolalde is primaryKey2 used?

Reply:
code that came from treatment schedule, no longer needed